CMSgov/design-system
# [WNMGDS-1330] idle timeout documentation

## Summary

Documentation for the Idle Timeout component.

## How to test

If you want to see in use, run `yarn storybook` and navigate to Components > Idle Timeout
